docs/usage/configuration-options.md

```diff
@@ -2972,7 +2972,7 @@ Table with options:
 | `gomodTidyE`                 | Run `go mod tidy -e` after Go module updates.                                                                                                              |
 | `gomodUpdateImportPaths`     | Update source import paths on major module updates, using [mod](https://github.com/marwan-at-work/mod).                                                    |
 | `helmUpdateSubChartArchives` | Update subchart archives in the `/charts` folder.                                                                                                          |
-| `npmDedupe`                  | Run `npm dedupe` after `package-lock.json` updates.                                                                                                        |
+| `npmDedupe`                  | Run `npm install` with `--prefer-dedupe` option. Only supported for NPM >= 7.                                                                                                        |
```

Comment:
```suggestion
| `npmDedupe`                  | Run `npm install` with `--prefer-dedupe` option. Needs npm >= 7.                                                                                                        |
```

The `npm` project styles their name in lowercase. We can simplify a bit by saying `needs x`.